src/main/java/edu/cornell/kfs/core/framework/persistence/ojb/CuStatementManager.java

```diff
@@ -0,0 +1,30 @@
+package edu.cornell.kfs.core.framework.persistence.ojb;
```

Comment:
Is there a standard convention for extending OJB libraries like this?  I wonder if the class should be more similar to what is being extended.  Something like edu.cornell.kfs.core.ojb.broker.accesslayer

Reply:
KualiCo placed many of their custom OJB-related classes under the `org.kuali.kfs.core.framework.persistence.ojb` package, so I was trying to use a CU-specific variant of that.

Reply:
Thanks, if KualiCo has a standard approach, I am fine with complying with that